canvas/models/models.py

```diff
@@ -166,10 +166,16 @@ def name(self):
     ("CHALLENGE", "CHALLENGE"),
 ]
 
+CHALLENGE_TYPE_CHOICES = [
+    ("QUOTA", "QUOTA"),
+    ("TOP_3_TEAMS", "TOP_3_TEAMS"),
```

Comment:
Let's have `TOP_TEAMS` as the challenge type and we can have a number showing the number of the fields as a separate field. In that case, we are not limited to only 3 teams.